vshard/replicaset.lua

```diff
@@ -222,6 +365,7 @@ local function replicaset_connect_to_replica(replicaset, replica)
         })
         conn.replica = replica
         conn.replicaset = replicaset
+        conn_vconnect_set(conn)
```

Comment:
Why do you need to add it here? It is not connected yet anyway. And on connect the vconnect will be added in the code above.